pkg/model/stack_serializer.go

```diff
@@ -405,17 +405,19 @@ func (serviceRaw *ServiceRaw) ToService(svcName string, stack *Stack) (*Service,
 	}
 	svc.NodeSelector = serviceRaw.NodeSelector
 
+	if svc.Labels == nil {
+		svc.Labels = make(Labels)
+	}
+	for key, value := range unmarshalLabels(serviceRaw.Labels, serviceRaw.Deploy) {
+		svc.Labels[key] = value
+	}
+
 	if stack.IsCompose {
 		if len(serviceRaw.Args.Values) > 0 {
 			return nil, fmt.Errorf("unsupported field for services.%s: 'args'", svcName)
 		}
 		svc.Entrypoint.Values = serviceRaw.Entrypoint.Values
 		svc.Command.Values = serviceRaw.Command.Values
-
-		for key, value := range unmarshalLabels(serviceRaw.Labels, serviceRaw.Deploy) {
```

Comment:
in the serializer we were converting `labels` into `annotations` - I think the model should reflect the manifest, so labels -> labels, annotations -> annotations.

Then, in the deployer to k8s, we can translate them into each other.

Reply:
Hmmm, I think that can be confusing. I don't think the model should represent always what is present in the manifest. We have some other things we change in the serializer. That's the idea of the serializer. Read the manifest and adapt it to whatever model we need internally. Propagate the difference between labels and annotations in compose to another layer might generate confusion. Also, I think you are changing more things here. If we translate a stack (not a compose), the labels have to be labels and annotations have to be annotations. Take a look to the else branch starting in line 419. Doing this change you are doing, we are inverting the meaning of labels and annotations for stacks, and that is not right. Could you review it, please?

Independently of where we end up having the translation from `labels` to `annotations` and viceversa, let's add a comment in the code, please. It is easy to be confused in several months, specially if someone without context in the CLI passes by that code.